src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Samples.cshtml

```diff
@@ -0,0 +1,292 @@
+@page "{id?}/{revisionId?}"
+@model APIViewWeb.Pages.Assemblies.UsageSamplePageModel
+@{
+    Layout = "ReviewLayout";
+    ViewData["Title"] = "Usage Samples";
+}
+<link rel="stylesheet" href="~/usagesample.css">
+
+@if (Model.Sample.SampleId == "Bad Deployment") // If the samples container does not exist
+{
+    <div class="container-fluid">
+        <div class="mx-5 mt-2">
+            <div class="text-muted">
+                <p> <strong> Usage Samples are not available for this APIView deployment. </strong></p>
+                <p> Please check the new DOCUMENTING.md to see how to enable them. </p>
+            </div>
+        </div>
+    </div>
+}
+else if (Model.Sample.SampleId == "File Content Missing") // if the file is broken in some way
+{
+    <div class="container-fluid">
+        <div class="mx-5 mt-2">
+            <div class="text-muted">
+                <p class="text-danger"> <strong> Bad blob read.</strong></p>
+                    @* Case 1 - There are other revisions on the server (Thus bad file read) *@
+                    @if(Model.SampleRevisions.Any()) {
+                        <p> However, revisions were found on the server. Please select any but the current one. </p>
```

Comment:
Is this description text to the user clear and correct? It feels odd that the first text is "However"

Reply:
It follows immediately from a notice about the blob read being bad, I believe it is clear but can change it if desired

Reply:
```suggestion
                        <p> The selected revision of this usage sample failed load. Please select another revision. </p>
```